src/components/client/EditClientViewModel.js

```diff
@@ -19,6 +19,7 @@ export function getNewClient(config) {
     attachments: [],
     extraFields: config.defaultExtraClientFields.slice(),
     defaultExtraInvoiceFields: config.defaultExtraClientInvoiceFields.slice(),
+    createdOn: new Date().getTime()
```

Comment:
is er een specifieke reden om die getTime() te doen? Een Date lijkt me een gepaster data type dan een number?